rpm/osg-pki-tools.spec

```diff
@@ -1,17 +1,16 @@
 Summary: osg-pki-tools
 Name: osg-pki-tools
-Version: 3.4.0
+Version: 3.5.0
 Release: 1%{?dist}
 Source: osg-pki-tools-%{version}.tar.gz
 License: Apache 2.0
 Group: Grid
 URL: http://github.com/opensciencegrid/osg-pki-tools
 BuildArch: noarch
-Requires: python
 Requires: m2crypto
```

Comment:
Does this bring in the Python3 m2crypto libs? @mtru32 is also interested in which package to install

Reply:
Ah right.  No, m2crypto is for python2.

The package is called python36-m2crypto on el7, which provides python3-m2crypto.  On el8, the package is just called python3-m2crypto.

Updated.

Reply:
And, @mtru32, as you can see in the changes for this PR, i'm just requiring `python3-m2crypto` (rather than `python36-m2crypto` separately for el7), since that single name will satisfy the requirement for both distros.

Reply:
Cool, thanks for keeping me up to speed on this @edquist